src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -484,9 +536,103 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             case ToolbarItemName.RemoveAllRegions:
                 this.canvas.current.confirmRemoveAllRegions();
                 break;
+            case ToolbarItemName.ActiveLearning:
+                await this.predict();
+                break;
+        }
+    }
+
+    private predict = async () => {
+        if (this.model && this.model.loaded) {
+            try {
+                const canvas = document.querySelector("canvas");
+                canvas.toBlob(async (blob) => {
+                    const imageBuffer = await new Response(blob).arrayBuffer();
+                    const buffer = Buffer.from(imageBuffer);
+                    const image64 = btoa(buffer.reduce((data, byte) => data + String.fromCharCode(byte), ""));
+                    const image = document.createElement("img");
+
+                    image.onload = async () => {
+                        this.predictImage(image);
+                    };
+                    image.src = "data:image;base64," + image64;
+                });
+            // tslint:disable-next-line:no-empty
+            } catch (_) {
```

Comment:
just curious on the empty catch; does this fire if the image fails to load or ?  Would it make sense to log or is this an anticipated/deterministic failures?

Reply:
In my test, specially with the WebApp, there are some strange issues where some of the internal calls my throw an exception in rare case.  For example even enabling the CORS server configuration and forcing crossOrigin="anonymous" in img/video object some times the canvas.getBlob just throw an exception.

IMHO auto detection (aka active learning) is an "utility" feature and in this particular case I don't think we should bored the user with an alert but we should simply do not detect nothing and warn no message at all.  But this is just my opinion here.  I don't know you but I hate windows like alert/messagebox and I prefer to use it only when strictly necessary.